get-nix-hashes.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
```

Comment:
Your scripts might use some simplification. Here is similar script I wrote for homebrew https://github.com/TarasMazepa/homebrew-stax/blob/103c53f17a515c1017e12c9df3347855bc43031c/.github/workflows/update.yml#L50